modules/aws/vpc/outputs.tf

```diff
@@ -42,23 +42,19 @@ output "aws_elb_console_id" {
   value = "${aws_elb.console.id}"
 }
 
-output "aws_elb_tnc_id" {
-  value = "${aws_elb.tnc.0.id}"
-}
-
 output "aws_lbs" {
-  value = ["${compact(concat(aws_elb.api_internal.*.id, list(aws_elb.console.id), aws_elb.api_external.*.id, aws_elb.tnc.*.id))}"]
+  value = ["${compact(concat(aws_elb.api_internal.*.id, list(aws_elb.console.id), aws_elb.api_external.*.id))}"]
 }
 
-output "aws_api_external_dns_name" {
+output "aws_elb_api_external_dns_name" {
```

Comment:
nit: I don't know if we want to handle this incrementally or not, but `aws_` is redundant for output variables in this AWS-only module.  Also, I don't know if consumers will care if this is an elastic load balancer or not.  So personally I'd prefer `api_external_dns_name` here.

But it's also nice to keep this diff small and the variables consistent, and `aws_elb_api_external_...` matches the pattern set by `aws_elb_api_external_zone_id` below.

Reply:
> But it's also nice to keep this diff small and the variables consistent, and `aws_elb_api_external_...` matches the pattern set by `aws_elb_api_external_zone_id` below.

That was my intention. Now it's consistent within the current scheme, which can be refactored followiing up to this change.